kivymd/uix/tab.py

```diff
@@ -720,40 +788,81 @@ class MDTabsBase(Widget):
     and defaults to `None`.
     """
 
+    def _get_label_font_style(self):
+        if self.tab_label:
+            return self.tab_label.font_style
+
+    def _set_label_font_style(self, value):
+        if self.tab_label:
+            if value in theme_font_styles:
```

Comment:
The documentation stated that was the proper use to register and read a new font. Let me test it in a different demo to see if that works, if not, a new issue must be opened

Reply:
As is usual, the problem is actually in how you state the question!

In this code - you should not be trying to register or read a *new* font. You are trying to read the fonts and actually font_styles that *have been* registered with the app.

If you use the current code - you will *only* get the standard fonts provided out of the box. If the app developer has followed the procedure as documented to add a new font to their app - it will be ignored by your code. This is obviously a bug in this code. I do not see the need for a new issue.

Reply:
The code only uses what's stored in memory.

Reply:
That is misdirection. It is all stored in memory. The question is whether you are using the correct values in the correct memory.